Models/Pokemon.cs

```diff
@@ -0,0 +1,9 @@
+namespace Bootcamp_API.Models;
+
+public class Pokemon
+{
+    public int id { get; set; }
```

Comment:
.NET uses PascalCase public properties

Reply:
I have changed the properties to be PascalCase.